### ISSUE TITLE
Q: How to test a successful build for all platforms?

### ISSUE THREAD

**Issue by lobingera** [JuliaPackaging/BinaryBuilder.jl#432]
I just wondered, let's assume we have a build_tarballs.jl that executes successfully on all (major) platforms. A lot of libraries have a 'make test' but that usually assumes a native run and not a cross run. Should this 'just' be done on the package side with CI? And what would be the feedback path to the builder?

**Comment by staticfloat** [comment]
Yes, we pretty much only do this on the package side.  We don't have any provision for tests or a direct feedback path, which is non-ideal.

I think the next step should be to make it easy to deploy a JLL locally, and use it from within the Julia package locally to make sure everything is on the up-and-up.
